app/assets/javascripts/router.js

```diff
@@ -396,9 +392,7 @@ class ReactRouter extends React.Component<Props> {
                 path="/auth/finishResetPassword"
                 render={({ location }: ContextRouter) => {
                   const params = Utils.getUrlParamsObjectFromString(location.search);
-                  const resetToken =
-                    typeof params.token === "string" ? decodeURI(params.token) : "";
```

Comment:
decodeURI wasn't necessary?

Reply:
The `getUrlParamsObjectFromString` utils functions already takes care of that by using `decodeURIComponent` (which is the correct function to use) on each key and value :)